profiles/coreos/base/package.accept_keywords

```diff
@@ -113,3 +113,6 @@ dev-util/checkbashisms
 
 # https://bugs.gentoo.org/show_bug.cgi?id=548158
 =sys-apps/gentoo-functions-0.10
+
+# Avoid cross compile error with amd64 stable (elfutils-0.158).
+dev-libs/elfutils ~amd64
```

Comment:
please use `=dev-libs/elfutils-0.1XX` (whatever the working version is) so we can eventually drop this rule.